snip.sh

```diff
@@ -24,11 +24,17 @@ __snip__replace_snips() {
   local prefix_tmp
   prefix_tmp=$(mktemp)
   local i=0
+ 
+  mkdir -p ~/.snip/cache/
```

Comment:
I am thinking about putting the cache files in `~/.cache/snip` instead. Agree?

Reply:
Ah yeah, we can do that